src/lib/components/homepage-variations/variation-config.ts

```diff
@@ -14,23 +17,62 @@ export interface HomepageVariationConfig {
     showPricing?: boolean;
 }
 
+const baseConfig: Partial<HomepageVariationConfig> = {
+    showDashboard: true,
+    showPlatforms: true,
+    showLogoList: true,
+    showBento: true,
+    showPullquote: true,
+    showCaseStudies: true,
+    showFeatures: true,
+    showMap: true,
+    showScale: true,
+    showPricing: true
+};
+
 // Predefined configurations for common variations
 export const variationConfigs: Record<string, HomepageVariationConfig> = {
```

Comment:
lets make a stricter type instead of `string`. feel free to export like `VariationKey = 'all-in-one-cloud' | '...' | '...';`, etc.